.github/workflows/docker-push-to-registries.yml

```diff
@@ -184,12 +196,11 @@ jobs:
 
   scan-image:
     needs: build
-    uses: Telicent-oss/shared-workflows/.github/workflows/trivy-image-scan.yml@main
+    uses: telicent-oss/shared-workflows/.github/workflows/trivy-image-scan.yml@update/for-frontend-trivy
```

Comment:
Do you mean for this to be a branch and not main?

Reply:
Yeah needs pinning back to `main` before merging